package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "lint": "eslint src",
     "check-types": "tsc --noEmit true",
+    "specs": "uvu -r ts-node/register tests",
```

Comment:
Could we use `jest` since that's used across all of the other bundlephobia's packages, and would make contributions easier.

Reply:
Done. Just thought that we could use something more lightweight for our tiny project. Jest is kinda gigantic)